packages/main/src/TableVirtualizer.ts

```diff
@@ -133,7 +133,7 @@ class TableVirtualizer extends UI5Element implements ITableFeature {
 	onTableAfterRendering(table: Table): void {
 		if (!this._table) {
 			this._table = table;
-			this._scrollContainer.addEventListener("scroll", this._onScrollBound, { passive: true });
+			this._scrollContainer?.addEventListener("scroll", this._onScrollBound, { passive: true });
```

Comment:
I do not get this Nayden how something in the shadow dom of the table might not be existing in the onAfterRendering of the table? what is the point of the onAfterRendering then?
If this is really happening in cypress tests then we should better have a look why is this happening.

Reply:
This is the onAfterRending of table component, not to TableVirtualzier

Reply:
this is obvious, the name of the method indicates this already.